src/WTG.Analyzers.TestFramework/TestData/SampleDataSet.cs

```diff
@@ -93,7 +96,7 @@ static SampleDataSet GetSampleData(Assembly assembly, string name, IEnumerable<K
 						break;
 
 					case "Diagnostics.xml":
-						LoadResults(assembly, pair.Value, ref languageVersion, ref diagnostics, ref suppressedIds, ref platforms);
+						LoadResults(assembly, pair.Value, ref languageVersion, ref diagnostics, ref suppressedIds, ref platforms, ref omitAssemblyReferences);
```

Comment:
Not too keen on the growing list of 'ref' arguments. Was planning to [refactor](https://github.com/WiseTechGlobal/WTG.Analyzers/commit/cb813aae9853d98a5abb64d2e876f12ea8728f39#diff-94dbc83679673f31408eb39b448fa7fd8fdd2e49a44390ab1d9c3032a772e9fc) this a little.

Reply:
yeah I kinda felt a bit dirty adding one more. Do you want it refactored upfront or should we keep that separate?